src/components/item-categories/styled.js

```diff
@@ -0,0 +1,27 @@
+import styled from 'styled-components';
+import { Rating } from 'react-rainbow-components';
+
+export const ResultItemRating = styled(Rating)`
+    height: auto;
+    display: block;
+    margin-bottom: 6px;
+`;
+
+export const ResultItemRatingLabel = styled.div`
+    font-size: 14px;
+    line-height: 1em;
+`;
```

Comment:
I see that ResultItemCategoriesWrapper is the only one used by ItemCategories, then if ResultItemRating and ResultItemRatingLabel are not being used remove them